features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/DefaultImageWireframeHelper.kt

```diff
@@ -36,6 +39,71 @@ internal class DefaultImageWireframeHelper(
     private val imageTypeResolver: ImageTypeResolver
 ) : ImageWireframeHelper {
 
+    @Suppress("ReturnCount", "LongMethod")
+    @UiThread
+    override fun createImageWireframe(
```

Comment:
I'm not so much in favor of this method overloading because I feel it duplicates part of the logic and also could be confusing when reading the code. If we want to go by this then I would suggest different method names and extract the common logic

Reply:
I agree with you, but actually the common parts of these two functions are : 
* creating placeholder if should be masked
* creating image wireframe
* resolve resource id asynchronously.
Each part calls another function which also has some tiny differences in parameters, so it's kind of hard to extract the common logic.

Does it sound good to you if I rename the function to `createImageWireframeByBitmap` and `createImageWireframeByDrawable`?

Reply:
functions names updated